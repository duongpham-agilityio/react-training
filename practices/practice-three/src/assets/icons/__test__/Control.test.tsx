import { render } from '@testing-library/react';

// Icons
import { Increase, Decrease } from '..';

describe('Decrease', () => {
  it('Match to snapshot', () => {
    const { container } = render(<Decrease />);

    expect(container).toMatchSnapshot();
  });
});

describe('Increase', () => {
  it('Match to snapshot', () => {
    const { container } = render(<Increase />);

    expect(container).toMatchSnapshot();
  });
});
