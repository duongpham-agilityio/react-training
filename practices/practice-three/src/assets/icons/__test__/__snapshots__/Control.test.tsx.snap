// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Decrease Match to snapshot 1`] = `
<div>
  <svg
    fill="#000"
    height="16"
    viewBox="0 0 16 16"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m3.86 8.753 5.482 4.796c.646.566 1.658.106 1.658-.753V3.204a1 1 0 0 0-1.659-.753l-5.48 4.796a1 1 0 0 0 0 1.506z"
    />
  </svg>
</div>
`;

exports[`Increase Match to snapshot 1`] = `
<div>
  <svg
    fill="#000"
    height="16"
    viewBox="0 0 16 16"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="m12.14 8.753-5.482 4.796c-.646.566-1.658.106-1.658-.753V3.204a1 1 0 0 1 1.659-.753l5.48 4.796a1 1 0 0 1 0 1.506z"
    />
  </svg>
</div>
`;
